package.json

```diff
@@ -11,19 +11,18 @@
   },
   "homepage": "https://github.com/Universal-Connect-Project/ucp-hosted-app#readme",
   "scripts": {
-    "build": "turbo build",
+    "build": "",
     "dev": "turbo dev",
     "lint": "turbo lint",
     "prettier": "turbo run prettier",
-    "test": "turbo run test",
-    "prepare": "husky"
```

Comment:
Why remove this?

Reply:
Same here, initially I thought this was causing build problems but I just found out it's not the case.